src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +391,46 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Prepare shown languages, source all available languages from the addons themselves
+	std::vector<std::string> languages_available;
+	for(const auto& a : addons_) {
+		for (const auto& b : a.second.locales) {
+			languages_available.push_back(b);
+		}
+	}
+	std::sort(languages_available.begin(), languages_available.end());
+	auto last = std::unique(languages_available.begin(),languages_available.end());
+	languages_available.erase(last, languages_available.end());
+
+	// Erase pt value, since otherwise Portugues do Brasil will be shown twice in the language list
+	languages_available.erase(std::remove(languages_available.begin(), languages_available.end(), "pt"), languages_available.end());
```

Comment:
Is removing this correct? There is both a "pt" and a "pt_BR" language code.

Reply:
Yes, this is intentional, since the already built in function `langcode_to_string` will only give out a valid translation for Portugues do Brasil. For the current state of localisation concerning language codes (pt, de, etc.) to language strings (Portugues, Deutsch etc.) available over that interface it should be correct. 

If one would refactor the `langcode_to_string` function that line would have to go, if it would provide a correct mapping for language codes to language strings